backend/src/config/common.js

```diff
@@ -0,0 +1,14 @@
+var common = {};
```

Comment:
plz change name common.js, we can name it more meaningful such as user_scores or user_reputation